samples/bluetooth/direct_test_mode/src/dtm.h

```diff
@@ -15,358 +16,246 @@
 extern "C" {
 #endif
 
-/* DTM command parameter: Upper bits mask. */
-#define LE_UPPER_BITS_MASK 0x0C
+/** @brief DTM PHY mode */
+enum dtm_phy {
```

Comment:
Yes, because nrf_radio_mode_t is compiled conditionally, so every enum would be present. The dtm.c file should be able to get (unfiltered) phy parameter and then decide if it's compatible or not.